cocos/ui/UITileImageView.h

```diff
@@ -0,0 +1,147 @@
+/****************************************************************************
+Copyright (c) 2013-2017 Chukong Technologies Inc.
+
+http://www.cocos2d-x.org
+
+Permission is hereby granted, free of charge, to any person obtaining a copy
+of this software and associated documentation files (the "Software"), to deal
+in the Software without restriction, including without limitation the rights
+to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+copies of the Software, and to permit persons to whom the Software is
+furnished to do so, subject to the following conditions:
+
+The above copyright notice and this permission notice shall be included in
+all copies or substantial portions of the Software.
+
+THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+THE SOFTWARE.
+****************************************************************************/
+
+#ifndef __UITILEIMAGEVIEW_H__
+#define __UITILEIMAGEVIEW_H__
+
+#include "2d/CCSprite.h"
+#include "ui/UIWidget.h"
+#include "ui/GUIExport.h"
+
+/**
+ * @addtogroup ui
+ * @{
+ */
+NS_CC_BEGIN
+
+struct CC_DLL ResourceData;
+
+namespace ui {
+/**
+ * @brief A widget to display images.
+ */
+class CC_GUI_DLL TileImageView : public Widget
+{
+    
+    DECLARE_CLASS_GUI_INFO
+    
+public:
+    /**
+     * Default constructor
+     * @js ctor
+     * @lua new
+     */
+    TileImageView();
+
+    /**
+     * Default destructor
+     * @js NA
+     * @lua NA
+     */
+    virtual ~TileImageView();
+
+    /**
+     * Create a empty ImageView.
```

Comment:
the comment is same as `UIImageView.h`, suggest to distinct both, for example change `Create a empty ImageView` to `Create a empty Tile ImageView`, other comments is similar.